src/session.rs

```diff
@@ -166,6 +166,61 @@ impl Session {
         )
     }
 
+    /// Constructs a new [`Command`] for launching subsystem `program` on the remote
+    /// host.
+    ///
+    /// Unlike [`command`](Session::command), this method does not shell-escape `program`, so it may be evaluated in
+    /// unforeseen ways by the remote shell.
+    ///
+    /// The returned `Command` is a builder, with the following default configuration:
+    ///
+    /// * No arguments to the program
+    /// * Empty stdin and dsicard stdout/stderr for `spawn` or `status`, but create output pipes for
+    ///   `output`
+    ///
+    /// Builder methods are provided to change these defaults and otherwise configure the process.
+    ///
+    /// ## Sftp subsystem
+    ///
+    /// To use sftp subsystem, it is recommended to enable feature `openssh-sftp-client`,
```

Comment:
There is no such feature as far as I can tell?

Reply:
Also, we should probably link to the `openssh-sftp-client` crate here.

Reply:
Doc updated!